src/modules/rtpengine/rtpengine.c

```diff
@@ -4545,9 +4546,21 @@ static void parse_call_stats_1(struct minmax_mos_label_stats *mmls,
 					(char *)stream->child->iov[1].iov_base);
 			LM_DBG("rtpengine: XXX stream child val type %i\n",
 					stream->child->sibling->type);
-			if((ssrc = bencode_dictionary_get_integer(stream, "SSRC", -1))
-					== -1)
-				continue;
+			ssrc = bencode_dictionary_get_integer(stream, "SSRC", -1);
+			if(ssrc == -1) {
+				ingress_ssrcs = bencode_dictionary_get_expect(
+						stream, "ingress SSRCs", BENCODE_LIST);
+				if(!ingress_ssrcs || !ingress_ssrcs->child)
+					continue;
+				LM_DBG("rtpengine: XXX got ingress SSRCs\n");
```

Comment:
Is this here intentionally or as a leftover?

Reply:
I'm not sure I fully understand the question... but this is here intentionally to make sure that ingress_ssrcs exists and has at least 1 element.

and the log message to indicate the affirmative

Reply:
so in the case there is no SSRC but an "ingress SSRC" the log looks as follows
```
rtpengine: XXX got ingress SSRCs
rtpengine: found SSRC '1778154774' for label 'Bleg_label'
```

Reply:
I also noticed that. Generally we should try to have log messages that are understandable and have some meaning. I don't get the meaning of this XXX in this case. ;-)

Reply:
all the log messages in the method have XXX in them... I was simply matching the existing code